src/random.cpp

```diff
@@ -14,16 +14,14 @@
 #include <wincrypt.h>
 #endif
 #include <logging.h>  // for LogPrintf()
+#include <randomenv.h>
+#include <support/allocators/secure.h>
```

Comment:
fa0e5b89cf7 should these be moved to `random.h`?

Reply:
Why?

Reply:
(or perhaps those are unrelated cleanups and I'm confused)

Reply:
The secure allocator is only needed for `g_rng`, which is in the cpp file. The lines are simply moved up a few lines. They have been sorted because I touched this file with my editor for other reasons.

Reply:
Gotcha, thanks